Checkout/Checkout/Source/Model/Country.swift

```diff
@@ -284,6 +284,15 @@ public struct Country: Equatable {
       return nil
     }
   }
+    
+    public init?(dialingCode: String) {
```

Comment:
So if merchant when initialising  Country  object with `+` you are removing it ?